.github/workflows/build-extensions.yml

```diff
@@ -676,3 +676,72 @@ jobs:
         with:
           name: WasmEdge-plugin-wasi_llm-${{ needs.get_version.outputs.version }}-${{ matrix.darwin_version }}_${{ matrix.arch }}.tar.gz
           path: plugin_wasi_llm.tar.gz
+
+  build_macos_metal:
+    permissions:
+      contents: write
+    strategy:
+      matrix:
+        include:
+          - system: MacOS 14 (arm64)
+            host_runner: macos-14
+            darwin_version: darwin_23
+            build_type: Release
+            arch: arm64
+    name: Plugins (Metal, ${{ matrix.system }}, clang++, ${{ matrix.build_type }})
+    runs-on: ${{ matrix.host_runner }}
+    env:
+      output_prefix: build/plugins
+      test_prefix: build/test/plugins
+      build_options: -DWASMEDGE_PLUGIN_STABLEDIFFUSION=ON -DWASMEDGE_PLUGIN_STABLEDIFFUSION_METAL=ON
+      tar_names: wasmedge_stablediffusion
+      test_bins: wasmedgeStableDiffusionTests
+      output_bins: libwasmedgePluginWasmEdgeStableDiffusion.dylib
+    needs: [ get_version ]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Build and install dependencies
+        run: |
+          eval $(/opt/homebrew/bin/brew shellenv)
+          brew install llvm@16 ninja cmake openssl opencv rust ffmpeg@6
```

Comment:
I don't think we need `opencv`, `openssl`, and `rust` for the stable diffusion plugin.

Reply:
removed